# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/client/framework/source-map-url-service.js
# Commit: 6eaa4d12b58c
# Full Hash: 6eaa4d12b58cb4762729fdf53511450926383db8
# Author: Nicolas Chevobbe <nchevobbe@mozilla.com>
# Date: 2020-10-15 09:40:06
# Regressor Bug: 1633712
# File Overlap Count: 1
# Description:
#   Bug 1633712 - [devtools] Create WorkerTargets as soon as possible. r=ochameau,devtools-backward-compat-reviewers.
#   
#   This patch adds support for dedicated worker targets in the Watcher actor.
#   Shared and Service workers are not handled yet.
#   
# ==============================================================================

diff -r 10f97f47c883 -r 6eaa4d12b58c devtools/client/framework/source-map-url-service.js
--- a/devtools/client/framework/source-map-url-service.js	Thu Oct 15 05:11:58 2020 +0000
+++ b/devtools/client/framework/source-map-url-service.js	Thu Oct 15 05:17:52 2020 +0000
@@ -416,6 +416,7 @@
     if (!this._prefValue) {
       return null;
     }
+
     if (this._target.isWorkerTarget) {
       return;
     }